classes/core/PKPApplication.inc.php

```diff
@@ -931,6 +965,41 @@ public static function getMetadataFields()
             'citations',
         ];
     }
+
+    /**
+     * Retrieves whether the application is installed
+     */
+    public static function isInstalled(): bool
+    {
+        return !!Config::getVar('general', 'installed');
+    }
+
+    /**
+     * Retrieves whether the application is running an upgrade
+     */
+    public static function isUpgrading(): bool
+    {
+        return defined('RUNNING_UPGRADE');
+    }
+
+    /**
+     * Retrieves whether the application is ready (installed and not being upgraded)
+     */
+    public static function isReady(): bool
```

Comment:
I just noticed this helper function and I like it! Can I make a naming suggestion? I feel like `isReady()` could mean a lot of things (ie - "is ready to be installed"). Can we call it something like `isRunning()` or `isActive()` or `isLive()`?

Or we could go the opposite route and prepare for a maintenance mode by renaming it to `isUnderMaintenance()` and returning the opposite `bool`.

Reply:
Sure! `isUnderMaintenance()` sound better for me =]

Reply:
Don't forget this change, @jonasraoni!